server/monitor/src/main/java/org/apache/accumulo/monitor/view/WebViews.java

```diff
@@ -80,7 +78,7 @@
   private void addExternalResources(Map<String,Object> model) {
     AccumuloConfiguration conf = monitor.getContext().getConfiguration();
     String resourcesProperty = conf.get(Property.MONITOR_RESOURCES_EXTERNAL);
-    if (isEmpty(resourcesProperty)) {
+    if (resourcesProperty.isBlank()) {
```

Comment:
Can this be null? I don't think so, because it has a default value, but not sure.

Reply:
I'm pretty sure it can't be null.